homeassistant/components/pvoutput/sensor.py

```diff
@@ -44,8 +44,8 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Required(CONF_API_KEY): cv.string,
-        vol.Required(CONF_SYSTEM_ID): cv.string,
+        vol.Required(CONF_API_KEY): cv.template,
```

Comment:
I don't think we should support templates here. That's not in line with our standards. This integration should be refactored to use a 3rd party library for the API interaction and a config flow for configuration.

Reply:
Agree, but for now we need a fix. The integration uses `homeassistant.components.rest.data` which wants templates for headers, so we need a temporary work-a-round for that.